lib/universal_id/portable_hash.rb

```diff
@@ -41,6 +42,15 @@ def transform(value, options:)
 
       value
     end
+
+    def implements_gid?(value)
+      value.respond_to?(:to_gid_param) && value.try(:persisted?)
```

Comment:
Do we want to force a `persisted?` check here? What about other object types, _that aren't ActiveRecords,_ that implement GlobalID? Do we want to support those?

Reply:
Right. I wanted to take this step by step - ultimately we need to support _new_ ActiveRecords. 

But it makes sense to remove this nonetheless 👍🏻